control-plane/gateways/deployment.go

```diff
@@ -69,6 +70,7 @@ func (b *meshGatewayBuilder) deploymentSpec() (*appsv1.DeploymentSpec, error) {
 				Labels: b.Labels(),
 				Annotations: map[string]string{
 					"consul.hashicorp.com/mesh-inject": "false",
+					constants.AnnotationGatewayKind:    "mesh",
```

Comment:
should "mesh" also be declared as a constant?

Reply:
done!